rs.ruffle.Ruffle.yaml

```diff
@@ -0,0 +1,65 @@
+app-id: rs.ruffle.Ruffle
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+#sdk-extensions:
+#  -  org.freedesktop.Sdk.Extension.rust-stable
+command: ruffle
+
+finish-args:
+  - --socket=wayland
+# <x11
+  - --socket=x11 # required for arboard fault
```

Comment:
I assume you meant `fallback-x11`  here?

Reply:
Nope! Ruffle has a hard dependency on x11 due to a currently unpatchable unsolvable fault in arboard

Reply:
If it works strictly only on X11, there's no point in allowing Wayland socket.

Reply:
*but*, it still renders with wayland. The backend library just needs to spawn an x11 window to momentarily work.